pkg/controller/admissionchecks/multikueue/workload.go

```diff
@@ -198,11 +198,11 @@ func (w *wlReconciler) Reconcile(ctx context.Context, req reconcile.Request) (re
 	}
 
 	managed, unmanagedReason, err := adapter.IsJobManagedByKueue(ctx, w.client, types.NamespacedName{Name: owner.Name, Namespace: wl.Namespace})
-	if err != nil {
+	if err != nil && !isDeleted {
```

Comment:
If `isDeleted=true`, then this `if` and the `if` below is never entered. Do we need to call the `IsJobManagedByKueue` at all if  `isDeleted=true`? 

If we do something line: 
```
if !isDeleted {
  // the current code
}
```

It would also help with the issue that this line may silence the error from `IsJobManagedByKueue`,

Reply:
done